Shubham_Sharma/Profile-bootstrap/Profile.html

```diff
@@ -0,0 +1,196 @@
+          <div class="text-center">
+              <img src="./Assets/images/profile2.jpg" alt="Not available" class="rounded-circle">
+          </div>
+          <br>
+          <div class="text-center">
+            <p class="lead text-light">Who's this ?</p>
+            <br>
+            <p class="text-light">That's me, SHUBHAM SHARMA. I am an INTERN at CYBER GROUP INDIA Pvt. Ltd. I am on my way to become a FULL STACK DEVELOPER, and I can bet that I am going to be a good one too.</p>
+          </div>
+        </div>
+    </div>
+
+    <!--NAVBAR is here-->
+    <nav class="navbar navbar-expand-lg sticky-top navbar-dark bg-dark align-items-end mb-3">
+        <a class="navbar-brand" href="#">SHUBHAM</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+      
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav mr-auto">
+            <li class="nav-item">
+              <a class="nav-link" href="#academics">Academics <span class="sr-only">(current)</span></a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#techKnowledge">Technical Knowledge</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link" href="#projects">Projects</a>
+            </li>
+          </ul>
+        </div>
+    </nav>
+    <br>
+    <br>
+
+    <!--Academics-->
+    <h1 class="text-center my-3 display-3" id="academics">Academics</h1>
+    <div class="row">
+        <div class="col-sm-1">
+
+        </div>
+        <div class="col-sm-5 mb-3">
+            <div class="card bg-pic">
+                <div class="card-block text-center py-3">
+                    <h3 class="card-title">Xth</h3>
+                    <p class="card-text">Sarvada Modern Secondary School</p>
+                    <p class="card-text"><b>CBSE</b></p>
+                    <p class="card-text"><b>8.8 CGPA</b></p>
+                    <a href="http://smsschool.net/" class="btn btn-primary">About School</a>
+                </div>
+          </div>
+        </div>
+        <div class="col-sm-5 mb-3">
+            <div class="card bg-pic">
+                <div class="card-block text-center py-3">
+                    <h3 class="card-title">XIIth</h3>
+                    <p class="card-text"><b>Rukmani Devi Jaipuria Public School</b></p>
+                    <p class="card-text"><b>CBSE</b></p>
+                    <p class="card-text"><b>86.8 %</b></p>
+                    <a href="http://rdjps.co.in/" class="btn btn-primary">About School</a>
+                </div>
+          </div>
+        </div>
+        <div class="col-sm-1">
+
+        </div>
+    </div>
+
+    <div class="row mb-5">
+        <div class="col-sm-3">
```

Comment:
why empty div

Reply:
I am using bootstrap grid, so to center the elements, I am using this empty div and setting columns.